af/fastpath/fastpath/core.py

```diff
@@ -1251,6 +1251,22 @@ def score_signal(msm) -> dict:
     return scores
 
 
+def score_torsf(msm) -> dict:
+    """Calculate measurement scoring for Tor Snowflake
+    Returns a scores dict
+    """
+    # https://github.com/ooni/ooni.org/issues/772
+    scores = init_scores()
+    tk = msm.get("test_keys", {})
+    failure = tk.get("failure")
+    if failure:
+        scores["accuracy"] = 0.0
```

Comment:
I don't think you should be setting the accuracy to zero in the case of `failure`. The meaning of the `failure` key is to indicate that there is an instance of blocking of the protocol.

Reply:
Fixed